OpenEdXMobile/res/values/constants.xml

```diff
@@ -1,7 +1,7 @@
 <resources>
-    <string name="eula_file_link" translatable="false">file:///android_asset/EULA.html</string>
-    <string name="privacy_file_link" translatable="false">file:///android_asset/privacy_policy.html</string>
-    <string name="terms_file_link" translatable="false">file:///android_asset/terms_and_services.html</string>
+    <string name="eula_file_link" translatable="false">file:///android_asset/eula_en.html</string>
```

Comment:
This string should be removed from here as its spanish translation is not available. In default english should be picked.